### PR TITLE
New Guidelines for BioDocker 

### PR DESCRIPTION
**Pull request by ypriverol**
@BioDocker/contributors : 

These are the new README for the architecture of BioDocker. Some of the major changes:

- Rename of BioDocker to BioContainers as was agree in https://github.com/BioDocker/biodocker/issues/63

- Remove the dependency in Docker and increase the scope to general containers technologies. 

- We try to clarify more the architecture of BioContainers including how to request a container, how to deploy etc. 

- We finally included a containers-specs.md document that explain which information should be included in the container. 

Regards

### CODE REVIEW

**Comment by ypriverol** [comment]
Please before accepting this pull-request @bgruening @prvst @sauloal @hroest @timosachsenberg @julianu @lgatto @BioDocker/owners @BioDocker/contributors  can you have a look and comment the details.

**Comment by prvst** [comment]
I think this is good to go. waiting for others to review it

**Comment by sauloal** [comment]
Looks fine to me

**Comment by ypriverol** [comment]
@bgruening after you review the guidelines please accept the PR.

File: README.md [hunk 1]
```diff
@@ -118,6 +157,15 @@ You can check the [Docker](https://docs.docker.com/reference/builder/) documenta
 
 Once the container is ready you can get in touch with us so we can make the appropriate arrangements to make your container available to everyone in the community by giving an automated build system.
 
+### 3.3. How to create a rkt based Biocontainer?
```
Comment (bgruening) [inline README.md]: Do we want to advertise this? quay.io supports both containers as soon as you deploy your container on there service. You get the conversion for Docker -> rkt for free (with some limitations).

File: container-specs.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+We explain here the containers metadata, their meaning and use in BioContainers: 
+ 
+
+
+| Field                 | Description                                                                                                                | Example                                                  | 
+|-----------------------|----------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------|
+| software version      | Every Biocontainer should contains the actual version of the software and tool                                             | software version: 2015020                                | 
+| software              | Name of the software                                                                                                       | software: Comet                                          |
+| description           | A short description of the software or tool.                                                                               | description: Peptide identification search tool          |
```
Comment (bgruening) [inline container-specs.md]: This makes all a lot of sense for regular images but less sense for mulled based ones. For example the version and the name is encoded in the namespace, the base-image is always `busybox` and there is no real maintainer.
That said we can think about adding these informations as yaml file into the container as `/usr/share/biocontainer_info.yaml`?

Also I would like to point out that mulled based containers are automatically build so all annotations we need to add manually would destroy one very nice feature of these containers :)

Here, I would like to propose to annotate the original recipe (conda, alpine, brew ...) and include the original recipe in the container as metadata.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -63,13 +63,13 @@ opened using [this form](https://github.com/BioDocker/biodocker/issues/new). Bio
...
-BioDocker's [BioDocker Specification and Design](http://github.com/BioDocker/biodocker/issues) is filled with comments and ideas
+BioContainers's [BioContainers Specification and Design](http://github.com/BioContainers/BioContainers/issues) is filled with comments and ideas
 for enhancements and we believe would make the best entry points for new developers.
 
 ## A Quick Note about Containers
 
-  For the most part, BioDocker containers should be published to the [BioDocker containers](https://github.com/BioDocker/biodocker) and not in this repository directly. 
-  If you are looking to supply new containers first check if an existing container exist in this repository [BioDocker containers](https://github.com/BioDocker/biodocker) -
+  For the most part, BioContainers containers should be published to the [BioContainers containers](https://github.com/BioContainers/BioContainers) and not in this repository directly. 
+  If you are looking to supply new containers first check if an existing container exist in this repository [BioContainers containers](https://github.com/BioContainers/BioContainers) -
```
Comment (timosachsenberg) [inline CONTRIBUTING.md]: exist -> exists

**Comment by timosachsenberg** [comment]
nice logo!

**Pull request merged**